test/build/50_dockerfile.bats

```diff
@@ -590,6 +591,10 @@ EOF
     fi
     prerequisites_ok argenv
 
+    sed_ () {
```

Comment:
I think explaining this function in a comment would be good since the regex is a bit intimidating.